scripts/next-dev-cycle.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# Default values
+DEV_MODE=${DEV_MODE:=false}
+MAIN_BRANCH=${MAIN_BRANCH:=main}
+NEXT_VERSION=""
+
+Help()
+{
+   # Display Help
+   echo "This tool prepares the project to the next development cycle."
+   echo ""
+   echo "Syntax: next-dev-cycle [OPTIONS]"
+   echo "options:"
+   echo "h    Display this guide."
+   echo ""
+}
+
+while getopts "hn::s" option; do
```

Comment:
What does the double-colon do?  (Also, what are the `n` and `s` options supposed to do?)

Reply:
leftover, they are not required at this moment